sdk/textanalytics/ai-text-analytics/samples/javascript/beginAnalyzeBatchActions.js

```diff
@@ -35,44 +35,49 @@ async function main() {
   };
   const poller = await client.beginAnalyzeBatchActions(documents, actions);
   const resultPages = await poller.pollUntilDone();
-
-  for await (const page of resultPages) {
-    const keyPhrasesResults = page.extractKeyPhrasesResults[0];
-    for (const doc of keyPhrasesResults) {
-      console.log(`- Document ${doc.id}`);
-      if (!doc.error) {
-        console.log("\tKey phrases:");
-        for (const phrase of doc.keyPhrases) {
-          console.log(`\t- ${phrase}`);
+  for (const page of resultPages) {
+    const keyPhrasesAction = page.extractKeyPhrasesResults[0];
+    if (!keyPhrasesAction.error) {
```

Comment:
User could intertest in how to handle the error when `keyPhrasesAction.error` is true

Reply:
It is pretty simple but it does not hurt to add it I guess. Probably the downside is showing failures in a sample to our customers does not sound like a good experience? Maybe I can add it in another sample other than the main one.